src/listeners/issue-embedder.ts

```diff
@@ -0,0 +1,28 @@
+import { Message } from 'discord.js';
+
+import { getIssueNumbers } from '../utilities/get-issue-numbers';
+import { issueEmbedGenerator } from '../utilities/issue-embed-generator';
+import { logger } from '../utilities/logger';
+import { getIssueData } from '../utilities/get-issue-data';
+
+export async function issueEmbedder(
+  message: Message,
+  autoLinkLimit: number
+): Promise<void> {
+  const issues = getIssueNumbers(message.content);
+  if (issues.length === 0) return;
+
+  const limtedIssues = issues.slice(0, autoLinkLimit);
+
+  for (const issueNumber of limtedIssues) {
+    try {
+      const { data } = await getIssueData(issueNumber);
+
+      const issueEmbed = issueEmbedGenerator(data, false);
+
+      message.channel.send(issueEmbed);
+    } catch (error) {
+      logger.error(error);
+    }
+  }
```

Comment:
This could end up putting the issues out of order depending on the `await` right? Please use `Promise.all` so they stay in the correct order.

Reply:
It won't put issues out of order with this codeblock because it's not an iterative array method.

Here's link explaining it -> https://stackoverflow.com/questions/37576685/using-async-await-with-a-foreach-loop